zos_concepts/zos_script/zos_script.yml

```diff
@@ -35,49 +39,114 @@
   vars:
     python_script_dir: "/u/user_name"
 
-    show_current_time: true
-    show_header: true
-    rexx_header: "#--- Current catalog information ---#"
+    use_custom_msg: True
+    custom_msg: "This is a custom Hello World message!!"
+    some_num: 3
+    fav_programming_languages:
+      - C
+      - rexx
+      - python
+      - cobol
 
   tasks:
-    # For this task, we can give arguments directly to the script by writing
-    # them after the command that we're interested in running on the managed
-    # node. Make sure to replace the values of the arguments.
-    - name: Run REXX script to get a job's information.
+
+  ##############################################################################
+  # This first block shows how a value can be passed directly to a script as a
+  # command line arg by using the 'cmd' module option.
+  # Notice in the output that the value of 'ansible_user' is present.
+  ##############################################################################
+
+  - name: Pass a value to a script in-line as a command line arg with the 'cmd'
+          module option and print the output.
+    tags: rexx_script
+    block:
+
+    - name: Run a Rexx script which prints a greeting to the user and prints the
+            current working directory on the remote system.
       ibm.ibm_zos_core.zos_script:
-        cmd: "{{ playbook_dir }}/files/JOB_INFO JOBID=ID OWNER=OWNER JOBNAME=NAME"
+        cmd: "{{ playbook_dir }}/files/HELLO_USER_PRINT_CWD.rexx name={{ ansible_user }}"
         remote_src: false
-      register: job_output
+      register: hello_user_output
 
-    - name: See Job information.
+    - name: Print the Rexx script output.
       ansible.builtin.debug:
-        msg: "{{ job_output }}"
+        msg: "{{ hello_user_output.stdout_lines }}"
+
+
+  ##############################################################################
+  # This next block explores module options: 'executable' and 'chdir'.
+  # The default behavior of the zos_script module is to run a script as Rexx.
+  # The task also passes in a command line arg like the previous one did.
+  # The 'executable' module option enables running other types of scripts by
+  # passing in a path to the appropriate executable. The Ansible inventory
+  # variable 'ansible_python_interpreter' is typically used to specify the
```

Comment:
isn't it always true instead of _tipically_?

Reply:
hmm, good point. I'll change it

My initial thinking was that the "typical" use was to tell ansible-core where remote python is, but since it's used in this playbook task to tell zos_script where remote python is, that would be an a-typical' use.